Sources/WinUIBackend/WinUIBackend.swift

```diff
@@ -1163,6 +1162,259 @@ public final class WinUIBackend: AppBackend {
         }
     }
 
+    public func createPathWidget() -> Widget {
+        WinUI.Path()
+    }
+
+    public func createPath() -> Path {
+        GeometryGroupHolder()
+    }
+
+    public func updatePath(_ path: Path, _ source: SwiftCrossUI.Path, pointsChanged: Bool) {
+        path.strokeStyle = source.strokeStyle
+
+        if pointsChanged {
+            path.group.children.clear()
+            applyActions(source.actions, to: path.group.children)
+        }
+
+        path.group.fillRule =
+            switch source.fillRule {
+                case .evenOdd:
+                    .evenOdd
+                case .winding:
+                    .nonzero
+            }
+    }
+
+    func requirePathFigure(
+        _ collection: WinUI.GeometryCollection,
+        lastPoint: Point
+    ) -> PathFigure {
+        var pathGeo: PathGeometry
+        if collection.size > 0,
+            let castedLast = collection.getAt(collection.size - 1) as? PathGeometry
+        {
+            pathGeo = castedLast
+        } else {
+            pathGeo = PathGeometry()
+            collection.append(pathGeo)
+        }
+
+        var figure: PathFigure
+        if pathGeo.figures.size > 0 {
+            // Note: the if check and force-unwrap is necessary. You can't do an `if let`
+            // here because PathFigureCollection uses unsigned integers for its indices so
+            // `size - 1` would underflow (causing a fatalError) if it's empty.
+            figure = pathGeo.figures.getAt(pathGeo.figures.size - 1)!
+        } else {
+            figure = PathFigure()
+            figure.startPoint = lastPoint
+            pathGeo.figures.append(figure)
+        }
+
+        return figure
+    }
+
+    func applyActions(_ actions: [SwiftCrossUI.Path.Action], to geometry: WinUI.GeometryCollection)
+    {
+        var lastPoint = Point(x: 0.0, y: 0.0)
+
+        for action in actions {
+            switch action {
+                case .moveTo(let point):
+                    lastPoint = Point(x: Float(point.x), y: Float(point.y))
+
+                    if geometry.size > 0,
+                        let pathGeo = geometry.getAt(geometry.size - 1) as? PathGeometry,
+                        pathGeo.figures.size > 0
+                    {
+                        let figure = pathGeo.figures.getAt(pathGeo.figures.size - 1)!
```

Comment:
Move this into the if condition to avoid the force unwrap

Reply:
The comment from ~30 lines earlier also applies here. Should I change both? I don't think the result can ever be nil in practice.